src/theme/palette.js

```diff
@@ -18,6 +18,21 @@ const palette = {
     800: '#1564EC',
     900: '#2242CD',
   },
+  green: {
+    light: '#4caf50',
+    main: '#388e3c',
+    dark: '#1b5e20',
+    50: '#e8f5e9',
+    100: '#c8e6c9',
+    200: '#a5d6a7',
+    300: '#81c784',
+    400: '#66bb6a',
+    500: '#4caf50',
+    600: '#43a047',
+    700: '#388e3c',
+    800: '#2e7d32',
+    900: '#1b5e20',
+  },
```

Comment:
Seems like the green color shade came from MUI's color palette. If we are using their predefined color palettes then I think we could directly import them instead of manually writing them down.